app/js/index.js

```diff
@@ -149,79 +147,8 @@ window.onload = function() {
   t.decorate(document.querySelector('#terminal'));
 
   t.onTerminalReady = function() {
+    UI_INSTANCE.OnHtermReady();
```

Comment:
@RandomlyKnighted  I'm really sorry that I still can't find your comment on this patch.
Could you please leave comment again?